airbyte-integrations/connectors/source-metabase/source_metabase/streams.py

```diff
@@ -25,14 +26,22 @@ def url_base(self) -> str:
         return self.instance_api_url
 
     def parse_response(self, response: requests.Response, **kwargs) -> Iterable[Mapping]:
-        response_json = response.json()
-        if self.response_entity:
-            result = response_json.get(self.response_entity, [])
-        else:
-            result = response_json
+        result = response.json()
         yield from result
 
 
+class IncrementalMetabaseStream(MetabaseStream, IncrementalMixin, ABC):
+    _cursor_value = None
+
+    @property
+    def state(self) -> Mapping[str, Any]:
+        return {self.cursor_field: str(self._cursor_value)}
```

Comment:
```suggestion
        return {self.cursor_field: str(self._cursor_value)}
```
Why are you casting the value to `str`? Don't you think it's risky for cursor value comparison?